core/src/tile/tileWorker.cpp

```diff
@@ -94,10 +97,10 @@ void TileWorker::run() {
 
             // float loadTime = (float(clock() - begin) / CLOCKS_PER_SEC) * 1000;
             // LOG("loadTime %s - %f", task->tile->getID().toString().c_str(), loadTime);
-
-            m_tileManager.tileProcessed(std::move(task));
```

Comment:
Yes, but this is good! :) What if the source does not have any data anymore - I mean the old tile that had data should be replaced by the empty tile.